ape_etherscan/verify.py

```diff
@@ -265,8 +265,9 @@ def attempt_verification(self):
             standard_json = json.dumps(standard_input_json, indent=2)
             logger.debug(f"{standard_json}\n")
 
-        # if there is a folder structure for source_id like xx/yy/zzz.sol
-        request_source_id = source_id.split("/")[-1]
+        # NOTE: Etherscan does not allow directory prefixes on the source ID.
+        request_source_id = Path(source_id).name
```

Comment:
the more i think about it - the more i wonder if this is a flaw on Etherscan's side.
im pretty sure source ids in ethpm stuff can have paths